app/models/organization.rb

```diff
@@ -1,6 +1,7 @@
 class Organization < ActiveRecord::Base
   attr_reader :verifier
   belongs_to :user
+  validates_length_of :organization_type, maximum: 5, allow_blank: true
```

Comment:
Prefer the new style validations `validates :column, length: value` over `validates_length_of`.